lib/manager/gradle/index.js

```diff
@@ -18,9 +18,10 @@ async function extractAllPackageFiles(config, packageFiles) {
   // Gradle needs all files to be written to disk before we parse any as some files may reference others
   // But if we're using gitFs then it's not necessary because they're already there
   if (!config.gitFs) {
-    for (const packageFile of packageFiles) {
-      const localFileName = path.join(config.localDir, packageFile);
-      const content = await platform.getFile(packageFile);
+    const files = [...packageFiles, 'gradlew'];
```

Comment:
Why do we need git-fs for gradlew? Is not easier just download the other 2 files that are required? I'm not against using git-fs if that makes it easier, but doesn't seem completely necessary right?

Reply:
See #3135: @rarkins told me that fetching files via API is deprecated and will be deprecated in favor of git-fs in the future - details are in the issue I believe.